Rubberduck.Inspections/InspectionResult.cs

```diff
@@ -108,7 +113,7 @@ public string ToClipboardString()
                 QualifiedSelection.Selection.StartLine);
         }
 
-        public virtual NavigateCodeEventArgs GetNavigationArgs()
+        public NavigateCodeEventArgs GetNavigationArgs()
         {
             return new NavigateCodeEventArgs(QualifiedSelection);
```

Comment:
Minor optimization: how about having a `private readonly Lazy<NavigateCodeEventArgs> _navigationArgs`  field set in the constructor, and have that property just return `_navigationArgs.Value`?